app/views/recipes/_recipe.html.erb

```diff
@@ -0,0 +1,8 @@
+
+   <p> <%= recipe.name %></p>
+  <div>
+    <p><%= link_to 'Delete', { action: :destroy, id: recipe.id }, method: :delete%></p>
+
+</div><p><strong>description</strong></p>
+    <%= recipe.description %>
+  
```

Comment:
- [ ] Please design your show.html.erb and this recipe partial to match this design: 
![image](https://user-images.githubusercontent.com/91486791/201356239-1c500ea7-fb8a-4271-b0f3-fc98b09d8cbd.png)